lib/autokey/gtkui/configwindow.py

```diff
@@ -62,15 +62,18 @@
 from .configwindow0 import get_ui
 
 
-def set_linkbutton(button, path):
+def set_linkbutton(button, path, filename_only=False):
     button.set_sensitive(True)
 
     if path.startswith(cm_constants.CONFIG_DEFAULT_FOLDER):
         text = path.replace(cm_constants.CONFIG_DEFAULT_FOLDER, _("(Default folder)"))
     else:
         text = path.replace(os.path.expanduser("~"), "~")
 
-    button.set_label(text)
+    if filename_only:
+        button.set_label(path[path.rindex("/")+1:])
```

Comment:
I feel like we should be using actual path parsing libraries here. This *might* do for now, but it smells awful. Change it if you can.

I suspect the fact that we aren't importing them may be a hangover from them not being in python2, perhaps.

Reply:
Not sure what you mean by "actual path parsing libraries" I have switched it to use `os.path.basename`:
```
    if filename_only:
        filename = os.path.basename(path)
        button.set_label(filename)
```

Which definitely looks much better

Reply:
Yup, I think that's what I had in mind; either that or create Path objects. This is indeed much better.